Main.hs

```diff
@@ -70,7 +70,7 @@ loop w checked color slider r = do
     True  -> putStrLn "Oh hi Mark"
     False -> return ()
 
-  sameLine >> arrowButton "Arrow" ImGuiDirUp
+  sameLine >> arrowButton "Arrow" ImGuiDir_Up
```

Comment:
A bit of a shame - do you think we should drop `_`, or have naming as exactly close to C as possible?

Reply:
My reasoning was that there are fields with double underscores (e.g. `ImGuiColorEditFlags__OptionsDefault`), and I thought it would be a bit confusing to now have those with single underscores and everything else with no underscores. I don't really mind though, if you prefer removing the first underscore I can do that.

Reply:
Ok, let's keep this how it is - that's a great point and I think it'll be more confusing if we change it.